bundles/org.openhab.binding.evcc/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,328 @@
+		<category>Energy</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="chargedEnergy">
+		<item-type>Number:Energy</item-type>
+		<label>Charged energy</label>
+		<description>Energy charged since plugged-in</description>
+		<category>Energy</category>
+		<state pattern="%.1f %unit%" readOnly="true"/>
+	</channel-type>
+	<channel-type id="charging">
+		<item-type>Switch</item-type>
+		<label>Charging state</label>
+		<description>Loadpoint is currently charging</description>
+		<category>Energy</category>
+		<state pattern="%f %unit%" readOnly="true">
+			<options>
+				<option value="ON">Charging</option>
+				<option value="OFF">Not charging</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="enabled">
+		<item-type>Switch</item-type>
+		<label>Charging enabled</label>
+		<description>Charging enabled (mode not "off")</description>
+		<category>Switch</category>
+		<state readOnly="true">
+			<options>
+				<option value="ON">Enabled</option>
+				<option value="OFF">Disabled</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="hasVehicle">
+		<item-type>Switch</item-type>
+		<label>Loadpoint has vehicle configuration</label>
+		<description>Whether vehicle is configured for loadpoint</description>
+		<category>Switch</category>
+		<state readOnly="true">
+			<options>
+				<option value="ON">Configured</option>
+				<option value="OFF">Not configured</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="maxCurrent">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Charging max current</label>
+		<description>Maximum amperage per connected phase with which the car should be charged</description>
+		<category>Energy</category>
+		<state min="0" step="1" pattern="%.0f %unit%" readOnly="false"/>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="minCurrent">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>Charging min current</label>
+		<description>Minimum amperage per connected phase with which the car should be charged</description>
+		<category>Energy</category>
+		<state min="0" step="1" pattern="%.0f %unit%" readOnly="false"/>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="minSoC">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Charging min SoC</label>
+		<description>Charge immediately with maximum power up to the defined SoC, if the charge mode is not set to "off"</description>
+		<category>Batterylevel</category>
+		<state min="0" step="1" max="100" pattern="%.0f %unit%" readOnly="false"/>
+		<autoUpdatePolicy>veto</autoUpdatePolicy>
+	</channel-type>
+	<channel-type id="mode">
+		<item-type>String</item-type>
+		<label>Charging mode</label>
+		<description>Charging mode: "off", "now", "minpv", "pv"</description>
+		<category>String</category>
+		<state readOnly="false">
+			<options>
+				<option value="off">Stop</option>
```

Comment:
Stop or Off?

Reply:
In the evcc UI it's called `Stop`, but off might be more appropriate.
(It switches the charging off).

What do you think?

Reply:
IMHO "Off" would be best as it best expresses state and action. "Stop" can only denote an action, whereas "Off" can mean "charging mode is **off**" (state) or "turn **off** charging mode" (action).